packages/ava/src/insight/algorithms/change-point/buishand-u.ts

```diff
@@ -13,7 +14,7 @@ export function buishandUTest(data: number[]): ChangePointItem & { uValue: numbe
   const meanValue = mean(data);
   const Sk = data.map((_, index) => sum(data.slice(0, index + 1)) - meanValue * (index + 1));
   const U = sumBy(Sk.slice(0, n - 1), (item) => item ** 2) / (n * (n + 1));
-  const Smax = maxabs(n, Sk, 1);
+  const Smax = maxabs(Sk);
```

Comment:
The names of Sk U and Smax are necessary?

Reply:
This is written in order to conform to the expression of the formula.
Do I need to modify it?

Reply:
just keep it